__tests__/app.test.js

```diff
@@ -0,0 +1,67 @@
+const request = require(`supertest`);
+const app = require(`../app.js`);
+const seed = require(`../db/seeds/seed.js`);
+const db = require(`../db/connection.js`);
+const testData = require(`../db/data/test-data/index.js`)
+const { TestWatcher } = require("jest");
```

Comment:
This might have been an auto import from VS Code - can be removed

Reply:
Done!